src/ChoiceList.re

```diff
@@ -32,41 +38,87 @@ let make = (_children) => {
       ReasonReact.Update({...state, label: Some(choice)})
       } : ReasonReact.Update({...state, label: None})
     | UpdateInput(s) => ReasonReact.Update({...state, input: s})
+    | Weissbier => ReasonReact.Update({...state, labels: ["Weissbier", ...state.labels]})
+    | UltimateWeissbier => ReasonReact.Update({...state, weissbier: changeThisBoolPlease(state.weissbier)})
     },
 
   render: self => {
     let new_label = self.state.input;
     let message =
-      "Add: " ++ new_label;
-    <div>
+      "Add";
+    let destinatorstring = "Destinator";
+    self.state.weissbier?
+    (<div>
+    <h2> (Util.str("Weissbier")) </h2>
+    <img src="/Users/expert239/Desktop/JSlearning/destinator-master/src/weissbier.png"/>
```

Comment:
_dang_, not gonna work

Reply:
Maybe just link to source URL?

Reply:
fixed with reference on src folder:
./src/weissbier.png

right?